src/views/choose-appeal-reason.njk

```diff
@@ -18,45 +19,48 @@
 {% endblock %}
 
 {% block content %}
-    <div class="govuk-width-container">
-        <main role="main" class="govuk-main-wrapper">
-            <div class="govuk-grid-row">
-                <div class="govuk-grid-column-two-thirds">
-                    <form method="post" id="choose-reasons-form">
-                        {{ 
-                            govukRadios({
-                                idPrefix: 'choose-reason',
-                                name: 'reason',
-                                fieldset: {
-                                    describedBy: 'choose-reason-hint',
-                                    legend: {
-                                        text: 'Why are you appealing this penalty?',
-                                        isPageHeading: true,
-                                        classes: 'govuk-fieldset__legend--xl'
-                                    }
-                                },
-                                hint: {
-                                    text: 'You can add more reasons later'
-                                },
-                                errorMessage: validationResult.getErrorForField('reason') if validationResult,
-                                items: [
-                                    { value: 'illness', text: 'Illness and health issues' },
-                                    { value: 'other', text: 'I’m appealing for another reason' }
-                                ]
-                            })
-                        }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {{
+                govukErrorSummary ({
```

Comment:
could you take a screenshot of the error summary banner and attach it to the description of the PR. This is usually a good idea when we are reviewing new pages

Reply:
Good idea, I'll do that now.